README.md

```diff
@@ -1 +1,46 @@
-# Sample-App-for-Postgres-for-TAS
+# Sample App for Postgres for TAS
+
+This repostory contains:
+
+1. Sample Cloud foundry Java Spring Webapp
+2. BBR restore script for restoring HA deployment's online S3 backup artifacts to any HA Postgres deployment
+
+
+## How to use the sample application
+
+### Pre-requisites
+
+* Java 8
+* Maven 3.0 or newer
+* Cloud Foundry CLI
+
+### 1. Build the application
+
+To build the application, run the following command from the root directory:
+
+```
+mvn clean install && mv target/postgres-demo-0.0.1-SNAPSHOT.jar ./
+```
+
+### 2. Deploy and bind the application
+
+To deploy the application, first you have to edit the `APP_NAME` and `SVC_INSTANCE_NAME` in the `manifest.ymml` file.
+Then run the following commands from the root directory:
+```
+cf push
+```
+This will create, push and bind the application to the service instance you mentioned in `manifest.yaml` file.
+
+
+## How to use the BBR restore script: ha-bbr-restore.sh
+
+This script is useful in only [this case](https://docs.vmware.com/en/VMware-Postgres-for-VMware-Tanzu-Application-Service/1.0/postgres/backup-restore.html#restoring-from-s3-backup-artifacts-3).
+The backup artifacts from HA service instance need a modification to be used with bbr restore. As you can see in the [directory structure for HA backup artifacts](https://docs.vmware.com/en/VMware-Postgres-for-VMware-Tanzu-Application-Service/1.0/postgres/backup-restore.html#structure-of-s3-backup-artifacts-2), there are 3 metadata files generated and saved, 1 for each postgres instance. To start your restore, first download the whole folder corresponding to your HA service deployment, as described in the structure section. BBR needs only 1 metadata file to be used during restore, which is essentially a union of all 3 files. The following script is helpful in doing this. Place it inside the top level service-instance_ folder downloaded from S3. After this, only keep the timetamp folders that you want. You can delete all other timestamp folders from all VM instance folders. Then:
+
+```
+cp ./ha-bbr-restore.sh donloaded-s3-backup-foder/
+cd downloaded-s3-backup-foder/
+./ha-bbr-restore.sh
+```
+
+This will create a new metadata file `metadata.json` which is required for BBR restore. Then this will run [regular BBR restore](https://docs.cloudfoundry.org/bbr/index.html#workflow-restore).
```

Comment:
How do we use the metadata.json? I looked at the link provided and there is no mention of a metadata file

Reply:
The customers shouldn't be concerned about metadata.json. How BBR uses it: its a manifest of whoch backup files map to which instance VM and whats their SHA sum.
Should we remove its mention altogether?

Reply:
I think we we should document what the user needs to do in order to use BBR. If that means that the user has to do something with this metadata file, then we need to provide better instructions because the link provided does not mention how to use a metadata file.

Reply:
Right. The customer doesn't need to do anything. The script will take care of calling BBR with correct args. So then I guess I should remove its mention completely.

Reply:
Done